app/views/devise/verify_authy.html.erb

```diff
@@ -19,3 +19,19 @@
   <%= authy_request_sms_link %>
   <%= submit_tag I18n.t('submit_token', {:scope => 'devise'}), :class => 'btn' %>
 <% end %>
+
+<% if @onetouch_uuid %>
+    <script>
+      (function(){
+        setInterval(function(){
+          var onetouchRequest = new XMLHttpRequest();
+          onetouchRequest.addEventListener("load", function(){
+            if(this.status != 202) clearInterval(onetouch_status);
```

Comment:
I'm getting an error about `onetouch_status` is not defined, that error cause that multiple approval requests are created, can you review it?, please

Reply:
Should be fixed